code/game/objects/structures/grille.dm

```diff
@@ -158,6 +158,8 @@
 			icon_state = "grille"
 			R.use(1)
 			return
+	else if(istype(W, /obj/item/weapon/rcd) && istype(loc, /turf/simulated)) //Do not attack the grille if the user is holding an RCD

```

Comment:
You can change the attack() proc of the RCD itself to prevent attacking the grille with it, same for windows, or anything, really.